src/check_oldies/forget_me_not.py

```diff
@@ -253,6 +259,18 @@ def get_parser():
             f"Defaults to {CONFIGURATION_FILE} if it exists."
         ),
     )
+    parser.add_argument(
+        "--user",
+        help=(
+            f"Username of the SMTP server."
+        ),
+    )
+    parser.add_argument(
+        "--password",
+        help=(
+            f"Password of the SMTP server."
+        ),
+    )
```

Comment:
It seems like a bad idea to have a secret in a command line, can you provide another way to achieve this?